app/views/users/invite.html.erb

```diff
@@ -48,10 +48,12 @@
       <%= error_message_on(f.object.errors, :roles) if @form.errors.key?(:roles) %>
       <fieldset class="govuk-fieldset">
         <div class="govuk-checkboxes" id="invite_user_form_roles">
-          <div class="govuk-checkboxes__item">
```

Comment:
Same argument as above

Reply:
I dont think its worth creating partials for this single occurrence, its only two options?  Same as you below comment. Its simple enough to do but would just create more complexity in understanding what the code does

Reply:
Good 👍🏾